### PR TITLE
[Model] Speed up Qwen2.5 VL by processing batched images separately by image encoder

### PR DESCRIPTION
**Pull request by kwang3939**
# Description

Unbatch batched images to be processed one single image at a time by image encoder.
Co-authored with @hfan.

# Tests
Test using MMMU dataset with lm-eval-harness: accuracy of 0.5178.
CI: https://buildkite.com/tpu-commons/tpu-inference-ci/builds/4030

# Checklist

Before submitting this PR, please make sure:
- I have performed a self-review of my code.
- I have necessary comments in my code, particularly in hard-to-understand areas.
- I have made or will make corresponding changes to any relevant documentation.

### CODE REVIEW

File: tpu_inference/models/jax/qwen2_5_vl.py [hunk 1]
```diff
@@ -778,7 +778,17 @@ def _process_image_input(
...
+            current_idx = 0
+            for i, image_thw in enumerate(grid_thw):
+                t, h, w = image_thw
+                image_size = t * h * w
+                end_idx = current_idx + image_size
+                image_pixel_values = pixel_values[current_idx:end_idx, :]
+                image_embeds.append(
+                    self.visual(image_pixel_values, (image_thw, )))
+                current_idx = end_idx
+            image_embeds = jnp.concatenate(image_embeds, axis=0)
```
Comment (hfan) [inline tpu_inference/models/jax/qwen2_5_vl.py:791]: This can be a follow up PR to make this improvement land first. But here it concats the embeddings of images, then the code below splits it back into individual image's embeddings again, we can avoid it.

**Comment by hfan** [comment]
I ran with lm_eval batch_size=auto, it got 51.89% MMMU: https://paste.googleplex.com/5346996089126912. It's the same as before, so I think we're good in terms of model quality
